README.md

```diff
@@ -2,161 +2,132 @@
+    - will be used as tag to base a GitHub Release on. A GitHub Release is just     a zip-file containing all files in the repository at that moment. The repository is setup in such a way that with each release a Zenodo archive will be created as well. The added benefit of this is (1) guaranteed long-term archiving, (2) creation of a DOI.
 
-### From a new template
 
+Each time a merge commit is made to the master branch of the `protocols` repo, a 'mirror read-only' repository (protocols-website) will be automatically triggered to build the rendered html versions of the protocols using Travis Continuous Integration.
+The resulting website is hosted at **TO BE ADDED**.
+This website will host all approved and published versions of all protocols.
 
-```r
-library(protocolhelper)
-create_sfp(title = "titel van het protocol",
-           subtitle = "optionele subtitel", 
-           short_title = "korte titel",
-           authors = "Voornaam Naam", 
-           date = "`r Sys.Date()`", 
-           reviewers = "Voornaam Naam, ...", 
-           file_manager = "Voornaam Naam", 
-           revision = "0.0.0.9000",
-           theme = "vegetation",
-           language = "nl",
-           from_docx = NULL,
-           protocol_number = NULL, 
-           render = TRUE)
-```
+## Fixed protocol features
 
+### Protocol-code
 
+A protocol-code consists of three characters and three digits separated by a dash.
 
-## How to contribute?
+|type             |theme      |theme_number |protocol-code
+|:----------------|:----------|:------------|:------------|
+|field            |generic    |0            |sfp-0##
+|field            |water      |1            |sfp-1##
+|field            |air        |2            |sfp-2##
+|field            |soil       |3            |sfp-3##
+|field            |vegetation |4            |sfp-4##
+|field            |species    |5            |sfp-5##
+|instrument       |           |             |sip-###
+|operating        |           |             |sop-###
+|activity         |           |             |sap-###
+|project          |           |             |spp-###
 
+The `##` indicates an incremental number. For instance, the first field protocol for "theme water" will have protocol code `sfp-101`.
 
-### Suggesting changes to existing protocols
+### Version number
 
-#### Using online editing functionalities
+The version number is of the form `YYYY.number`. `YYYY` indicates the year in which the protocol was released. The `number` indicates the order of release within that year.
 
-To do: write this section
+### Language identifier
 
+Protocols can be either written in one or multiple languages. The functions in the `protocolhelper` package support Dutch (language = 'nl') and English (language = 'en').
 
-#### Using RStudio and git version control
+Together with a version number, a language identifier and the protocol-code, a unique version of a protocol will be identified.
```

Comment:
It is not strictly necessary, but maybe the easiest thing to do. So I will change the text a bit. 

(Protocol-code + version number will be sufficient to uniquely identify a protocol if we assume that two language versions of the same version of a protocol will never be released at the same time (same merge commit). I think this is safe to assume.)

Reply:
> will be sufficient to uniquely identify a protocol if we assume that [...] will never be released at the same time (same merge commit)

Do you mean that the version number will be different between 'nl' and 'en', hence there is a distinction? I think we will have to decide whether we accept apparently evolved versions `sfp-401-2020.6` and `sfp-401-2020.7` which actually are each other's translations.

On the other hand, we won't have that so much. Perhaps an intermediate solution could be to add the language identifier when it is different from 'nl'.

Reply:
Good point!
Maybe the following could work?
````
protocols-2020.6   sfp-401-nl-2020.6 #a first nl version
protocols-2020.7   sfp-401-nl-2020.7 #a second nl version
protocols-2020.8   sfp-401-en-2020.6 #translation of nl to en of version 2020.6
protocols-2020.9   sfp-401-nl-2020.9 #a third nl version (bump to 2020.9 instead of 2020.8)
`````

Reply:
I understand the idea, however I'm afraid it would create confusion about the meaning of a version number. Also, it would complicate management (= more error-prone). I think there would be no problem with:

`protocols-2020.8   sfp-401-en-2020.8 #translation of nl to en of version 2020.6`

It is more consistent with the expectation that version numbers are identical between tags that point to the same commit. To get a sensible timeline of one protocol, one would then need to do it _language-specific_, but I think that makes sense. I.e. filtering by either `sfp-401-nl` or `sfp-401-en` to see what the consecutive versions are.

Considering the gaps in version numbers this creates, seen from a language-specific protocol, note that also ongoing releases of other protocol-codes, e.g. `sfp-101`, will cause gaps in the version numbers of `sfp-401` anyway.

I think there is another opportunity of keeping version numbers and language identifiers independent: it effectively allows `sfp-401-nl` and `sfp-401-en` to be (somewhat) different in their contents as well, perhaps because of collaboration with other researchers, etc. This would make clear that both are actually different protocols, though still about the same subject and method.

Reply:
OK, I think we go with your idea. In that case, somewhere in the translated protocol it should be mentioned from which protocol it is translated from (i.e. a reference to the protocol specific tag).

Reply:
@ThierryO I'd like to hear your opinion on this too.

Reply:
I'd upgrade all language versions in the same PR. Hence they get the same version number. If you plan a translated version, use it to first update the current version. If you update the original version, then you should update the translations accordingly.

Reply:
That's probably the neatest way to do it. @florisvdh do you agree?
That means we can drop the language identifier from the specific tag.

Reply:
I've implemented this in 71fbb07

Reply:
Sure, we can go with this approach. It does narrow workflow flexibility and may request more effort from the contributor (enforcing parallel translation). On the other hand it makes the release cycle and protocol version numbering easier for the protocol user. Also, if content differs more than pure translation, this approach will imply a different protocol code, which makes that aspect more transparent.

Then, how will we cope with not-willingness to proceed with translating into a certain language, if an older version exists in that language? A reason may be that that translation is not longer needed at a certain point. For consistency, I suggest deleting it in the next development cycle; older versions remain on the website anyway. You may want to add such a step in the workflow, now that we've thought about it :wink:

Reply:
Will at that to the workflow and after that is done I will merge this branch to master. Thanks to both of you for your help!